pkg/acsengine/const.go

```diff
@@ -57,9 +57,9 @@ var KubeImages = map[api.OrchestratorVersion]map[string]string{
 		"exechealthz":  "exechealthz-amd64:1.2",
 		"addonresizer": "addon-resizer:1.6",
 		"heapster":     "heapster:v1.2.0",
-		"dns":          "kubedns-amd64:1.7",
+		"dns":          "k8s-dns-kube-dns-amd64:1.13.0",
```

Comment:
Should this also be updated for Kubernetes 1.6.2, 1.6.0, and prior versions? If not, then update the PR description & commit as to why for archiving/history.